src/android/dlt-logd-converter.cpp

```diff
@@ -50,8 +54,13 @@ static inline struct logger *init_logger(struct logger_list *logger_list, log_id
 
 static struct logger_list *init_logger_list(bool skip_binary_buffers)
 {
+#if __ANDROID_API__ >= 31
+    int mode = 0;
```

Comment:
Hello @sebastienraillet ,
can you please use "O_RDONLY" instead of the hardcoded "0"?

Reply:
Hello @michael-methner,
Yes sure 👍  In fact, I didn't see that previously, **ANDROID_LOG_RDONLY** was a define on **O_RDONLY**.
We can even directly pass O_RDONLY in both case (before and after API 31) avoiding the #if / #endif.
Which solution do you prefer ?

Reply:
Hello @sebastienraillet 

For Android S and above, the path change: https://cs.android.com/android/platform/superproject/+/master:system/logging/liblog/include/log/
Hence please correct Android blue print file: 
https://github.com/COVESA/dlt-daemon/blob/master/Android.bp#L198

For backward compatibility, I still have no idea. If anybody has good suggestions please comment!